packages/block-editor/src/components/rich-text/README.md

```diff
@@ -121,14 +121,22 @@ registerBlockType( /* ... */, {
 } );
 ```
 
+## RichText.isEmpty
+
+Helper function to test if supplied variable (e.g. content of RichText) is empty. It tests if value of variable is empty or if variable is an empty array.
+
+### `value: String | Array`
+
+### Example
+
+Some [attributes in paragraph block](https://github.com/WordPress/gutenberg/blob/59ed45e2d9db32e50b17b5801f0841f6d7489b0c/packages/block-library/src/paragraph/edit.js#L174) are set differrently if content of `RichText` is empty.
```

Comment:
It's probably better to give an inline example as done in other places.

Reply:
Agreed.